### PR TITLE
Connected Research page with featured articles page

### PR DESCRIPTION
**Pull request by LemonFace0309**
dont forget to npm install

- added a disabled prop to the card and button shared component
image

- linked researched page via url with featured article page
- added social media icons with sharing capabilities in articles page
image

### CODE REVIEW

File: components/shared/Card.js [hunk 1]
```diff
@@ -16,11 +16,15 @@ import styles from './Card.module.css'
...
+ * @param {String} link - link to where the button leads
+ * @param {Boolean} disabled - true if button should be disabled
  * @param {React.Component} children - is the paragraph text
  */
-function AppCard({ title, img, children, btnText }) {
+function AppCard({ title, img, btnText, disabled, link, children }) {
   btnText = btnText ? btnText : 'READ MORE'
   title = title ? <Card.Title>{title}</Card.Title> : null
+  link = link ? link : '#'
+  disabled = disabled ? true : false
```
Comment (jimmyygit) [inline components/shared/Card.js:27]: lol? what's the point of this

File: pages/featured-articles/[slug].js [hunk 2]
```diff
@@ -33,6 +47,7 @@ function Article({ article }) {
                 <Image
                   src={getStrapiMedia(article.author.image)}
                   layout="fill"
+                  s
```
Comment (jimmyygit) [inline pages/featured-articles/[slug].js]: I think u left an `s` here

**Review by jimmyygit — APPROVED** [review]
image
Why does the `\n` char show? Is there a way to make a new line?

File: components/shared/Card.js (hunk 1 above)
Comment (jimmyygit) [reply components/shared/Card.js:27]: whoops. only wanted to highlight line 27

Comment (LemonFace0309) [reply components/shared/Card.js:27]: prop to add a special class to the button to show it's disabled

File: pages/featured-articles/[slug].js (hunk 2 above)
Comment (LemonFace0309) [reply pages/featured-articles/[slug].js]: > image
> Why does the `\n` char show? Is there a way to make a new line?

double space. /n was just testing

**Pull request merged**